docs/DeveloperGuide.adoc

```diff
@@ -297,15 +299,22 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
 |Priority |As a ... |I want to ... |So that I can...
 |`* * *` |new user |see usage instructions |refer to instructions when I forget how to use the App
 
-|`* * *` |user |add a new person |
+|`* * *` |project leader |add a project member | asssign tasks to them
+
+|`* * *` |project leader |add tasks |
 
-|`* * *` |user |delete a person |remove entries that I no longer need
+|`* * *` |project leader |change task status | ger reminder of the progress of each task
 
-|`* * *` |user |find a person by name |locate details of persons without having to go through the entire list
+|`* * *` |project leader |sync team members schedules |Find a time slot where the maximum number of people, if not all, can attend
+|`* *` |Project leader with many members and tasks|Find out which group members are working on a particular task |Distribute workload evenly
+|`* *` |Project leader with tasks that require inventory|assign tasks inventory or materials that are needed |Download an inventory report whenever required
 
-|`* *` |user |hide <<private-contact-detail,private contact details>> by default |minimize chance of someone else seeing them by accident
+|`* *`|Project leader with frequent meetings|Create a new meeting with recommended time and desired location|View my next meeting in the home page
```

Comment:
Probably can split the function and benefit into 2 user stories since it does not seems to match.
As a project leader, I want to create meeting so that I can secure the meeting slot at my desired time and place.
As a project leader, I want to view my next meeting in the homepage so that I know my schedule at a glance.